code/modules/hydroponics/compost_bin.dm

```diff
@@ -0,0 +1,192 @@
+#define BASE_BIOMASS_CAPACITY 1500
+#define BASE_COMPOST_CAPACITY 1500
+#define SOIL_COST 25
+#define SOIL /obj/item/stack/sheet/soil
+#define DECAY 0.2
+#define MIN_CONVERSION 10
```

Comment:
Could you `undef` these if you don't use them outside of this file?

Reply:
sure